yarn-project/aztec.js/package.json

```diff
@@ -54,13 +55,17 @@
     "@types/lodash.partition": "^4.6.0",
     "@types/lodash.zip": "^4.2.7",
     "@types/node": "^18.7.23",
+    "assert": "^2.1.0",
+    "browserify-zlib": "^0.2.0",
     "buffer": "^6.0.3",
     "crypto-browserify": "^3.12.0",
     "jest": "^29.5.0",
     "jest-mock-extended": "^3.0.3",
     "process": "^0.11.10",
+    "querystring-es3": "^0.2.1",
     "resolve-typescript-plugin": "^2.0.1",
     "stream-browserify": "^3.0.0",
+    "stream-http": "^3.2.0",
```

Comment:
From what you mentioned earlier, I understand these are the new deps needed to please the webpack god. It seems these are required because we now have a dependency on `aztec/ethereum`, which is only needed to grab the `L1ContractAddresses` type.

Maybe we can just move the `L1ContractAddresses` to `aztec/types`, so we don't introduce the new dependency (since we want js to be as slim as possible), and don't have these new issues with webpack?